assets/charts/control-plane/kubelet/templates/kubelet-ds.yaml

```diff
@@ -65,36 +65,50 @@ spec:
         securityContext:
           privileged: true
         volumeMounts:
+        # Used by Calico.
```

Comment:
I _think_ this might be needed only for `kubenet` network plugin BTW.

If this is "Used by Calico", then it shouldn't be here? If we add comments here, we should explain why kubelet needs it?

Is it used by Calico CNI scripts which run in kubelet FS namespace?

Reply:
Fixed this to be more specific.